### PR TITLE
add inital gcp service Compute Engine docs

### PR DESCRIPTION
**Pull request by mstumpfx**
Need review for mostly README.md as it will be a template for the rest of the GCP services

### CODE REVIEW

**Comment by asuresh4** [comment]
@mstumpfx the table under metrics section would created from the docs directory. So you could remove it from the readme. Both for the docs site and for the app.

File: google-compute-engine/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{ "display_name":"Google Compute Engine",
+  "description": "SignalFx's integration with Google Compute Engine",
+  "project_url": "https://github.com/signalfx/integrations/tree/master/google-computeengine",
+  "code": "https://cloud.google.com/compute",
+  "featured": false,
+  "logo_large": "integration_googlecomputeengine",
```
Comment (keitwb) [inline google-compute-engine/meta.yaml]: This needs .png at the end.  Images aren't working in lab.

**Comment by barbara-sfx** [comment]
@asuresh4   Can you look at the readme? I think the first part (or some part somewhere) should be telling them to go to the docs for installation instructions. e.g. https://docs.signalfx.com/en/latest/getting-started/send-data.html#connect-to-gcp  Am I correct about that?

**Comment by mstumpfx** [comment]
@barbara-sfx I was keeping it consistent with the way we do it with the AWS docs too; 
a single [AWS Integration](https://github.com/signalfx/integrations/tree/master/aws-elasticache) links back to the [AWS main repo](https://github.com/signalfx/integrations/tree/master/aws) which has a more descriptive way to install, and a link to the docs.

**Comment by barbara-sfx** [comment]
Oh ok, I see. Yeah, that makes sense.

**Pull request merged**